### PR TITLE
Mention dependency files update for Gradle

### LINKED ISSUE #10042: More precise description of dependabot's Gradle support
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#package-ecosystem

### What part(s) of the article would you like to see updated?

> [1] Dependabot doesn't run Gradle but supports updates to the following files: build.gradle and build.gradle.kts (for Kotlin projects).

This is no longer precise, Dependabot can also update e.g. `dependencies.gradle`.

### Additional information

https://github.com/dependabot/dependabot-core/pull/4155#issuecomment-918500360

### PR DESCRIPTION
**Pull request by zbynek**
### Why:

Closes #10042

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: data/reusables/dependabot/supported-package-managers.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ poetry         | `pip`            | v1               | | **✓** | |
 Terraform      | `terraform`      | >= 0.13, <= 1.0  | **✓** | **✓** | |
 yarn           | `npm`            | v1               | **✓** | **✓** | |
 
-[1] {% data variables.product.prodname_dependabot %} doesn't run Gradle but supports updates to the following files: `build.gradle` and `build.gradle.kts` (for Kotlin projects).
+[1] {% data variables.product.prodname_dependabot %} doesn't run Gradle but supports updates to the following files: `build.gradle`, `build.gradle.kts` (for Kotlin projects) and files included via the `apply` declaration that have `dependencies` in the filename.
```
Comment (asciimike) [inline data/reusables/dependabot/supported-package-managers.md:29]:
```suggestion
[1] {% data variables.product.prodname_dependabot %} doesn't run Gradle but supports updates to the following files: `build.gradle`, `build.gradle.kts` (for Kotlin projects) and files included via the `apply` declaration that have `dependencies` in the filename. Note that `apply` does not support `apply to`, recursion, or advanced syntaxes (e.g. Kotlin's `apply` with `mapOf`, filenames defined by property).
```

Comment (asciimike) [reply data/reusables/dependabot/supported-package-managers.md:29]: Adding caveats in https://github.com/dependabot/dependabot-core/pull/4155#issuecomment-918500360

**Comment by ramyaparimi** [comment]
@zbynek This PR seems to be opened as a draft by mistake. If so, please change it to `ready for review` so I can triage it for our team to review ✨

**Review by asciimike — DISMISSED** [review]

**Comment by zbynek** [comment]
@ramyaparimi thanks for pointing it out, I was waiting for the rendering ( https://docs-10041--patch-2.herokuapp.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#package-ecosystem ) and forgot to mark this as ready.

**Comment by ramyaparimi** [comment]
@zbynek 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: data/reusables/dependabot/supported-package-managers.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ poetry         | `pip`            | v1               | | **✓** | |
 Terraform      | `terraform`      | >= 0.13, <= 1.0  | **✓** | **✓** | |
 yarn           | `npm`            | v1               | **✓** | **✓** | |
 
-[1] {% data variables.product.prodname_dependabot %} doesn't run Gradle but supports updates to the following files: `build.gradle` and `build.gradle.kts` (for Kotlin projects).
+[1] {% data variables.product.prodname_dependabot %} doesn't run Gradle but supports updates to the following files: `build.gradle`, `build.gradle.kts` (for Kotlin projects) and files included via the `apply` declaration that have `dependencies` in the filename. Note that `apply` does not support `apply to`, recursion, or advanced syntaxes (e.g. Kotlin's `apply` with `mapOf`, filenames defined by property).
```
Comment (ethanpalm) [inline data/reusables/dependabot/supported-package-managers.md:29]:
```suggestion
[1] {% data variables.product.prodname_dependabot %} doesn't run Gradle but supports updates to the following files: `build.gradle`, `build.gradle.kts` (for Kotlin projects), and files included via the `apply` declaration that have `dependencies` in the filename. Note that `apply` does not support `apply to`, recursion, or advanced syntaxes (for example, Kotlin's `apply` with `mapOf`, filenames defined by property).
```

Just some small changes based on the GitHub docs style guide ⚡

**Review by ethanpalm — DISMISSED** [review]
👋🏼  Thanks for creating this PR, @zbynek! Your changes look great and should help users avoid confusion. I've suggested some very minor changes just to bring a sentence in line with our style guide. I'll update the branch and once all the checks are passing, I'll get this merged 🎉 

Thanks for contributing to the GitHub docs! ⚡

**Review by ethanpalm — APPROVED** [review]

**Comment by ethanpalm** [comment]
I'm in the process of getting this PR merged, but there have been some technical issues, so just need to make sure those are addressed and then this will be merged as soon as possible.

**Pull request merged**